src/main/java/com/scaleunlimited/flinkcrawler/topology/CrawlTopologyBuilder.java

```diff
@@ -405,9 +405,10 @@ public FetchUrl map(Tuple3<CrawlStateUrl, FetchUrl, FetchUrl> justHasFetchUrl)
         crawlDbIteration.closeWith(robotBlockedUrls.union(queuedStatusUrls, fetchStatusUrls, newUrls));
 
         // Save off parsed page content by passing it on to the provided content sink function.
-        parsedUrls
-            .name("Select fetched content")
-            .addSink(_contentSink)
+        DataStreamSink<ParsedUrl> content = parsedUrls
+            .addSink(_contentSink);
+
+        content = content
```

Comment:
Can't you just do .name() and .setParallelism() on the content stream above, without making this a separate assignment here?